frontstage/templates/surveys/surveys-upload-failure.html

```diff
@@ -11,17 +11,23 @@
 
         <div class="panel panel--error u-mb-s" data-qa="error">
             <div class="panel__header">
-                <h1 class="panel__title venus">Something went wrong</h1>
+                 {% if ".xlsx format" in error_info.text %}
```

Comment:
Would be better to have a designed error protocol between services, e.g a json-formatted response containing a code, with a translation function to map codes to text.